qa/zcash/create_wallet_200k_utxos.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python2
+# Copyright (c) 2017 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#
+# Create a large wallet
+#
+# To use:
+# - Copy to qa/rpc-tests/wallet_large.py
+# - Add wallet_large.py to RPC tests list
+# - ./qa/pull-tester/rpc-tests.sh wallet_large --noshutdown
```

Comment:
--nocleanup rather than --noshutdown ?

Reply:
I used `--noshutdown` because I wanted to double-check things afterwards, but yes it could be that for general usage.